### ISSUE TITLE
docker pull ubuntu:16.04 hangs on docker for Mac

### ISSUE THREAD

**Issue by harsimranmaan** [tianon/docker-brew-ubuntu-core#50]
I am using docker v1.11.1-beta10 for Mac . Pulling the latest/16.04 ubuntu image hangs as shown below. I am able to pull other images just fine. Any suggestions to debug?

```bash
harsimran$ docker -D pull ubuntu:16.04
16.04: Pulling from library/ubuntu
6d28225f8d96: Downloading [=>                                                 ]  1.48 MB/48.35 MB
166102ec41af: Download complete 
d09bfba2bd6a: Download complete 
c80dad39a6c0: Download complete 
```

**Comment by tianon** [comment]
I can't reproduce :disappointed:

```console
$ docker -D pull ubuntu:16.04
16.04: Pulling from library/ubuntu

6d28225f8d96: Already exists 
166102ec41af: Already exists 
d09bfba2bd6a: Already exists 
c80dad39a6c0: Already exists 
a3ed95caeb02: Already exists 
Digest: sha256:eb96613af7b0cd3839dcfb1e4184d94222d2f2127ed5bb4fcb047805ddcf56b6
Status: Image is up to date for ubuntu:16.04
```

There might be something hanging up in the networking of your VM, but I think the Docker for Mac folks would probably be more helpful in trying to debug that. :confused:

**Comment by harsimranmaan** [comment]
The issue was resolved after upgrading to docker v1.11.1-beta11. Don't know the root cause.
